cgo/libclang_config_llvm14.go

```diff
@@ -0,0 +1,16 @@
+//go:build !byollvm && llvm14
+// +build !byollvm,llvm14
+
+package cgo
+
+/*
+#cgo linux        CFLAGS:  -I/usr/lib/llvm-14/include
+#cgo darwin amd64 CFLAGS:  -I/usr/local/opt/llvm@14/include
+#cgo darwin arm64 CFLAGS:  -I/opt/homebrew/opt/llvm@14/include
+#cgo freebsd      CFLAGS:  -I/usr/local/llvm14/include
+#cgo linux        LDFLAGS: -L/usr/lib/llvm-14/lib -lclang
+#cgo darwin amd64 LDFLAGS: -L/usr/local/opt/llvm@14/lib -lclang -lffi
+#cgo darwin arm64 LDFLAGS: -L/opt/homebrew/opt/llvm@14/lib -lclang -lffi
```

Comment:
These are incorrect; cf #2608.

Reply:
Thanks, fixed! My branch was rather old and so didn't have that fix yet.